Add Overworld test plans (#112)
Gamify-IT/docs: dev-manuals/test-plans/overworld-frontend/plans/minigame-spot-configured-done.md

> # Name: Minigame spot configured and done

Reviewer: ```suggestion
# Minigames: Pre-game start (`u.minigames-1`)
```
(also notice that I changed the identifier to minigame**s**)

Number: U.minigame.1  
Version: V0.2, 2022-08-09, split test case
Author: Florian Wüst  
Tester: -  

## Description

> This use case verifies that loading the minigame and player data for a configurated minigame from the backend works as intended. Furthermore, player specific data like the highscore and the status of the minigame needs to be loaded. All that info has to be stored in the minigame object.
> 
> ## Precondition
> 
> The minigame object is configured in the overworld (world and index are set) and the needed data exists in the backend.  
> The game is started and the backend runs.
> 
> ## Postcondition
> 
> The minigame object is set up according to the data in the data base.

Reviewer: ```suggestion
This test walks the user through the procedure necessary to start a minigame. Specifically, it tests that minigame spots are colored correctly and that a popup is shown before the minigame can be started.

## Precondition

The minigame (spot) is configured in the overworld and its backend.  
The game is started and the backend runs.
Optionally, the backend already has statistics for this minigame and player.

## Postcondition

Nothing changed for the backend.
```

## Typical procedure

> 1. The tester walks to the minigame spot.  
> 2. The minigame spot is displayed and has a blue color.
> 3. The tester walks into the minigame spot.  
> 4. The minigame overview panel opens.
> 5. The displayed minigame and player specific highscore are correct as set in the backend.

Reviewer: ```suggestion
1. Walk to the minigame spot
2. The minigame spot is displayed
3. The minigame spot is colored blue
4. Walk **in**to the minigame spot
4. The minigame overview panel opens
5. The displayed minigame and player specific highscore are the ones stored in the backend
```

## Alternative procedures

> N/A

Reviewer: ```suggestion
3.1. The player already played the minigame but has not gotten the highscore - the minigame spot is red.
3.2. The player already achieved the highscore for the minigame - the minigame spot is yellow.
```
(Not sure about when what color is displayed, though)
